skipchain/skipchain.go

```diff
@@ -855,6 +894,16 @@ func (s *Service) forwardLinkLevel0(src, dst *SkipBlock) error {
 
 	// create the message we want to sign for this round
 	roster := src.Roster
+
+	if enableViewChange {
+		// If the server identities in the two rosters are the same,
+		// then it might be a view-change, so we use the second roster
+		// with the new leader.
+		if isRotation(src.Roster, dst.Roster) {
```

Comment:
I wonder if this should be a method on type onet.Roster? So that both sk and ol can use it and not duplicate code?

Reply:
made an issue for that, should be able to fix it quickly in a future PR https://github.com/dedis/cothority/issues/1362